.gitignore

```diff
@@ -1,3 +1,8 @@
+# Build subdirectories.
+/*build*
+!/build-aux
```

Comment:
Can we remove the other [build-aux](https://github.com/hebasto/bitcoin/pull/302/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R27-R39) references?

Reply:
It will be done in https://github.com/hebasto/bitcoin/pull/166.